presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableHandle.java

```diff
@@ -29,6 +29,7 @@ public class IcebergTableHandle
         extends BaseHiveTableHandle
 {
     private final IcebergTableName icebergTableName;
+    // TODO: this field is no longer useful, would be removed in a subsequent PR
```

Comment:
Shouldn't we just remove in this PR? Or will it create a very large change? I feel like it should just be removed.